rpcserver.go

```diff
@@ -3509,6 +3509,8 @@ func (r *rpcServer) marshallRoute(route *routing.Route) *lnrpc.Route {
 			Fee:              int64(fee.ToSatoshis()),
 			FeeMsat:          int64(fee),
 			Expiry:           uint32(hop.OutgoingTimeLock),
+			PubKey: hex.EncodeToString(
```

Comment:
hmm, strange wrapping?

Reply:
I thought so too, but this is what gofmt does.

Reply:
No